tests/phpunit/tests/post/mimeTypes.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Unit tests covering post mime types.
+ *
+ * @ticket 59195
+ *
+ * @group post
+ *
+ * @covers ::get_available_post_mime_types
+ */
+
+class Tests_Get_Available_Post_Mime_Types extends WP_UnitTestCase {
+
+	public function tear_down() {
+		// Remove all uploads.
+		$this->remove_added_uploads();
+		parent::tear_down();
+	}
+
+	/**
+	 * Test get_available_post_mime_types.
+	 */
+	public function test_get_available_post_mime_types() {
+		// Upload a JPEG image.
+		$filename = DIR_TESTDATA . '/images/test-image.jpg';
+		$contents = file_get_contents( $filename );
+		$upload   = wp_upload_bits( wp_basename( $filename ), null, $contents );
+		$this->assertEmpty( $upload['error'] );
+		$this->_make_attachment( $upload );
+
+		// Upload a PDF file.
+		$filename = DIR_TESTDATA . '/images/test-alpha.pdf';
+		$contents = file_get_contents( $filename );
+		$upload   = wp_upload_bits( wp_basename( $filename ), null, $contents );
+		$this->assertEmpty( $upload['error'] );
+		$this->_make_attachment( $upload );
+
+		$mime_types = get_available_post_mime_types();
+
+		$this->assertSame( array( 'image/jpeg', 'application/pdf' ), $mime_types );
```

Comment:
Assertions should probably have the `$message` parameter defined as recommended by the guidelines on https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions

Reply:
Thanks! Addressed in 3a4c4162f0.